types/akamai-edgeworkers/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "lib": [
             "es6"
         ],
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
Rather than globally suppressing this error, can you please suppress it in the source locally with a comment? Thank you!

Reply:
I've reverted the suppression. We're ready for a re-review.